### ISSUE TITLE
[Bug]: UNS Init Unable to Accept

### ISSUE THREAD

**Issue by alphabravo6** [amyinorbit/x-uns-bugs#22]
### Version

1.0.9

### What happened?

# What happened
Loaded into a flight, completed flows and checklists, and turned on all avionics, including the UNS 1 and 2. When on the init page, I tried clicking accept through L5 button, although nothing happened. I tried accepting through both L5 and clicking the 'enter' key, but nothing occurred. I then tried restarting both UNS' through the on/off dim menu, which didn't change anything. I also tried reloading the aircraft entirely, which also did nothing. I also tried realigning through clicking the 'ATT/HDG ALIGN' button, which also did nothing. 

# Expected behaviour
When accepting on the init page, it would be expected that the UNS would confirm/lock in the aircraft position, date, and UTC time. 

[Log.txt](https://github.com/amyinorbit/x-uns-bugs/files/7630323/Log.txt)

**Comment by alphabravo6** [comment]
I couldn't upload the video directly as it was over 10mb, but I have just uploaded it to vimeo instead: https://vimeo.com/656537156

**Comment by alphabravo6** [comment]
Any update on this??? I'm really itching to get back in the Q4XP :( @amyinorbit

**Comment by amyinorbit** [comment]
There's nothing clear in the log, but I'm trying to work on it. Please remember it's the holiday period, also.

**Comment by amyinorbit** [comment]
@alphabravo6 could you please delete the directory at `xp11/Output/caches/UNS-1` and try again? I think it might be caused by a corrupt airport database.

**Comment by alphabravo6** [comment]
Thanks for your reply! I just tried deleting the files under that address, although it wouldn't let me delete some as they were corrupt. These were the corrupt files that i couldn't delete :( 

![image](https://user-images.githubusercontent.com/49643600/148510968-ac58a66e-a311-4c4e-9f63-4cd6573fdfb1.png)

**Comment by amyinorbit** [comment]
@alphabravo6 there's definitely something wrong with the cache. Please delete the UNS-1 cache completely (force delete if necessary?) and check that the permissions are set correctly.

**Comment by alphabravo6** [comment]
Yep - not sure what's happening :( I have tried deleting the UNS-1 cache folder, although it still stays there. When I go into the folder and try and click on one of the long/lat region folders, I get this 
![image](https://user-images.githubusercontent.com/49643600/149603077-590acf82-9a67-44a5-885d-b19ed3f09893.png)

**Comment by alphabravo6** [comment]
In regards to checking permissions, how would I do that?

**Comment by alphabravo6** [comment]
After close to two months, I have fixed the issue. Through following method one on this microsoft help forum, I was able to delete the UNS-1 cache folder. 

Here is the link to the forum that I used as a guide. 

https://answers.microsoft.com/en-us/windows/forum/all/cannot-delete-a-corrupted-file-on-my-external-hard/b751bdd0-7703-49c1-8ab6-1e24dcae7b91

Thank you so much for putting up with me and this issue @amyinorbit haha. 

Back to flying the Dash...
